src/common/models/ProjectionsPair.ts

```diff
@@ -51,7 +52,12 @@ export class ProjectionsPair {
   }
 
   get locationName(): string {
-    return this.left.locationName;
+    const region = this.left.region;
+    if (region instanceof MetroArea) {
+      return `${region.fullName}, ${region.stateCodes}`;
+    } else {
+      return region.fullName;
+    }
```

Comment:
This fixes the labels in compare so that metros include their state code(s).  (i.e. "Ponce metro area, PR" instead of just "Ponce metro area")